src/styles/header.scss

```diff
@@ -0,0 +1,55 @@
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.logo {
+  display: inline;
+  width: $width;
+  margin-top: 10px;
+}
+
+.nav {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 9 point. Check this in all places

Reply:
You haven't fixed this. You should create a separate file for nav block